suite-native/formatters/src/components/PercentageDifferenceFormatter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { Text, TextProps } from '@suite-native/atoms';
+import { useFormatters } from '@suite-common/formatters';
+
+import { FormatterProps } from '../types';
+
+type NumberValues = number;
+type PercentageDifferenceFormatterProps = FormatterProps<NumberValues> & {
+    secondValue: NumberValues;
```

Comment:
I thinking about name `oldValue` or `baseValue` because you always comparing value agains some old or base value to get difference.